tests/test_repo.py

```diff
@@ -50,33 +71,65 @@ def test_new_files_object(self):
 
-    def test_set_target_branch(self):
+    def test__set_target_branch(self):
+        self.repo._set_target_branch('master')
+        self.assertEqual(self.repo.source_branch, 'refs/heads/master')
+
+    def test__set_target_branch_source_branch(self):
+        self.repo._set_target_branch('master', 'something')
+        self.assertEqual(self.repo.source_branch, 'refs/heads/something')
+
+    def test__set_target_branch_reset_file_cache(self):
+        self.repo._set_target_branch('master')
         cached_files = ['cached_file', 'cached_file', 'cached_file']
-        self.repo.files = cached_files.copy()
-        self.repo.set_target_branch('master')
+        self.repo.files = cached_files
         self.assertEqual(self.repo.files, cached_files)
 
-        self.repo.set_target_branch('Something different')
+        self.repo._set_target_branch('Something different')
         self.assertEqual(self.repo.files, [])
         self.assertEqual(self.repo.target_branch, 'Something different')
         self.assertEqual(self.repo.target_ref, 'refs/heads/Something different')
 
     def test_create_pr(self):
         repo = Repo(None, branch='', github=self.mock_git)
-        repo._original_repo = MagicMock()
-        repo._forked_repo = MagicMock()
-        repo._forked_repo.owner.login = 'someone'
+        repo._target_repo = MagicMock()
+        repo._source_repo = MagicMock()
+        repo._source_repo.owner.login = 'someone'
         repo.branch_name = 'branch'
         pr = repo.create_pr('test', '', 'target_branch', ['test'])
-        repo._original_repo.create_pull.assert_called_once_with('test', '', 'target_branch', 'someone:branch')
+        repo._target_repo.create_pull.assert_called_once_with('test', '', 'target_branch', 'someone:branch')
         pr.set_labels.assert_called_once_with('test')
-        repo._forked_repo.create_pull.assert_not_called()
+        repo._source_repo.create_pull.assert_not_called()
+
+    def test__get_new_version_major(self):
+        version_file = MagicMock()
+        version_file.decoded_content = '1.2.3'.encode('utf-8')
+        self.repo.version_file = version_file
+        self.repo.semver_label = 'major'
+        self.repo._get_new_version()
+        self.assertEqual(self.repo.new_version, '2.0.0')
+
+    def test__get_new_version_minor(self):
+        version_file = MagicMock()
+        version_file.decoded_content = '1.2.3'.encode('utf-8')
+        self.repo.version_file = version_file
+        self.repo.semver_label = 'minor'
+        self.repo._get_new_version()
+        self.assertEqual(self.repo.new_version, '1.3.0')
+
+    def test__get_new_version_patch(self):
+        version_file = MagicMock()
+        version_file.decoded_content = '1.2.3'.encode('utf-8')
+        self.repo.version_file = version_file
+        self.repo.semver_label = 'patch'
+        self.repo._get_new_version()
+        self.assertEqual(self.repo.new_version, '1.2.4')
```

Comment:
I realize these tests are totally unrelated to the PR, but I noticed we didn't have any tests around this method, so... 🤷‍♂️